### ISSUE TITLE
Linux DevTools: text is dark, background is same dark color, unreadable.

### ISSUE THREAD

**Issue by 13zebras** [typora/typora-issues#5373]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
The DevTools in Typora are unreadable as it appears the font color is the same as the background color. There is no way to see the text or use DevTools accordingly.

**To Reproduce**
1. Go to 'View' menu
2. Click on '.Toggle Devtools'
3. Devtools opens.
5. Impossible to use since nothing can be seen. (Photo attached)

**Screenshots / Screencasts**
Attached

**Sample Markdown File**
ALL .md FILES

**Desktop (please complete the following information):**
 - OS: Linux
 - Distro:  Endeavour OS (Arch Linux)
 - Kernel: 5.19.1-zen1-1-zen
 - i3 window manager
 - GTK3 theme: Dracula

I've tried dark and light GTK themes, nothing improves DevTools. 
Same with documents, themes in Typora.

**Typora Version**
Most recent release. 1.3.6

**Additional context**
I can access DevTools in other electron apps and see the text.

![2022-08-15_02-41-19_flameshot](https://user-images.githubusercontent.com/17152930/184603987-7c7fc4f4-00fc-4677-89a8-f190e6314a80.png)

**Comment by abnerlee** [comment]
Does a light Typora theme (like Github) make the devTool background white?

**Comment by 13zebras** [comment]
Every theme is the same. Dark text, dark background. I tried all the built-in ones.

**Comment by abnerlee** [comment]
Does 1.4.4 fix this?

**Comment by 13zebras** [comment]
Same. However, I spoke too soon above when I said I can access devtools in other electron apps. True, I can. But I cannot in ALL electron apps. I tried vscode, and no, I cannot see devtools. I could in a few others, but not vscode. Also, I haven't used obsidian in a while, so I tried it. Nothing, exactly the same.

In trying to figure this out, I have arrived at one possibility, all of which involve electron and gtk. I have gsettings set to always select dark theme, and this potentially could be causing some weird interaction with electron apps. I will check it out when I have a chance.
